heudiconv/__init__.py

```diff
@@ -2,13 +2,7 @@
 import logging
 import os
 from .info import __packagename__
-
-try:
-    from importlib.metadata import version
-except ImportError:
-    from importlib_metadata import version
-
-__version__ = version(__name__)
+from ._version import __version__
```

Comment:
I think it should just fall down to this if `__version__` is '0.0.0' ie not provided by versioningit, since I would not be getting `_version.__version__` automagically updating with every git commit and thus `heudiconv.__version__` even when I have versioningit (e.g. on my laptop) would not reflect actual version.

Reply:
`versioningit` doesn't update with every Git commit either, unless you reinstall after each commit, but reinstalling causes `_version.py` to be updated anyway.

Reply:
oh, right -- it relies on the generated PKG-INFO, so unlike versioneer it is not "dynamic"... that is a cons of versioningit then since I will see myself reporting "wrong" versions.  But then indeed this patch would be fine as far as versioningit goes I guess.